opensearchapi/api.indices.put_template.go

```diff
@@ -35,6 +35,7 @@ import (
 	"time"
 )
 
+// Deprecated: uses a nonexistent legacy REST API (/_template) correct REST API is /_index_template use newIndicesPutSettingsFunc instead
```

Comment:
Can you please update all the comments like following. Keep in mind to adjust to the correct function/type.
```suggestion
// Deprecated: uses legacy API (/_template), correct API is /_index_template, use newIndicesPutIndexTemplateFunc instead
```